src/XrdOuc/XrdOucPrivateUtils.hh

```diff
@@ -0,0 +1,58 @@
+//------------------------------------------------------------------------------
+// Copyright (c) 2024-2025 by European Organization for Nuclear Research (CERN)
+// Author: Cedric Caffy <cedric.caffy@cern.ch>
+//------------------------------------------------------------------------------
+// XRootD is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// XRootD is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with XRootD.  If not, see <http://www.gnu.org/licenses/>.
+//------------------------------------------------------------------------------
+
+
+/**
+ * PRIVATE HEADER for utility functions, implementation in XrdOucUtils.cc
+ */
+#ifndef XROOTD_XRDOUCPRIVATEUTILS_HH
+#define XROOTD_XRDOUCPRIVATEUTILS_HH
+
+#include <string>
+#include <vector>
+#include <regex>
+
+static const std::string OBFUSCATION_STR = "REDACTED";
+
+// As the compilation of the regexes when the std::regex object is constructed is expensive,
+// we initialize the auth obfuscation regexes only once in the XRootD process lifetime
+static const std::vector<std::regex> authObfuscationRegexes = {
```

Comment:
Shouldn't this go in the .cc file instead of in the header? Same for `"REDACTED"` above.

Reply:
That's absolutely right. Thought that it would be enough to declare and initialize them on the same file, but I'll modify. Thanks @amadio !

Reply:
Done